test/integration/test-extensions-loading.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {createFixtureIframe} from '../../testing/iframe';
+
+// Checks if an amp element gets upgraded.
+// Works for all amp elements used in this test.
+function checkElementUpgrade(element) {
+  expect(element).to.have.class('-amp-element');
+  expect(element).to.have.class('-amp-layout-responsive');
+  expect(element).to.have.class('-amp-layout-size-defined');
+  expect(element).to.not.have.class('-amp-notbuilt');
+  expect(element).to.not.have.class('amp-notbuilt');
+  expect(element).to.not.have.class('amp-unresolved');
+  expect(element).to.not.have.class('-amp-unresolved');
+}
+
+describe('test extensions loading in multiple orders', () => {
+  let fixture;
+  it('one extension, extension loads first, all scripts in header', () => {
+    return createFixtureIframe(
```

Comment:
All actual test bodies are the same, right?

Could you make a function like

```js
function testLoadOrderFixture(name) {
  …
}
```

that takes the fixture name and always runs the same assertions?